packages/admin/cms-admin/src/dam/DataGrid/FolderDataGrid.tsx

```diff
@@ -372,6 +376,24 @@ const FolderDataGrid = ({
             hideSortIcons: true,
             disableColumnMenu: true,
         },
+        {
+            field: "importSourceType",
+            headerName: intl.formatMessage({
+                id: "comet.dam.file.importSourceType",
+                defaultMessage: "Source",
+            }),
+            renderCell: ({ row }) => {
+                if (row.__typename === "DamFile") {
```

Comment:
nit: Everywhere else in this file the `isFile(row)` helper is used. Why not here?

Reply:
Because I didn't look at the rest of the file, sorry 😬 Fixed: 0cd30d465d6ec21a1ae8cc9d7f687fa37ee16c32